package.yaml

```diff
@@ -37,6 +37,8 @@ dependencies:
 
 default-extensions:
   - DerivingStrategies
+  - DerivingVia
+  - GeneralisedNewtypeDeriving
```

Comment:
`GeneralisedNewtypeDeriving` requires 8.6, but `GeneralizedNewtypeDeriving` goes all the way back to 6.8. I see no reason to use the British spelling.